messybrainz/db/release.py

```diff
@@ -285,3 +307,83 @@ def create_release_clusters():
         create_release_clusters_without_considering_anomalies,
         create_release_clusters_for_anomalies,
     )
+
+
+def fetch_releases_from_musicbrainz_db(connection, recording_mbid):
+    """ Fetches releases from the MusicBrainz database for the recording MBID
+        and returns a list of releases which consists a dict with release MBID 
+        and release name.
+    """
+
+    return mb_release.get_releases_using_recording_mbid(recording_mbid)
+
+
+def fetch_recording_mbids_not_in_recording_release_join(connection):
+    """ Fetches recording MBIDs that are present in recording_json table
+        but are not present in recording_release_join table and returns
+        a list of those recording MBIDs.
+    """
+
+    result = connection.execute(text("""
+        SELECT DISTINCT recj.data ->> 'recording_mbid'
+                   FROM recording_json AS recj
+              LEFT JOIN recording_release_join AS rrj
+                     ON (recj.data ->> 'recording_mbid')::uuid = rrj.recording_mbid
+                  WHERE recj.data ->> 'recording_mbid' IS NOT NULL
+                    AND rrj.recording_mbid IS NULL
+    """))
+
+    return [mbid[0] for mbid in result]
+
+
+def truncate_recording_release_join():
+    """Truncates the table recording_release_join."""
+
+    with db.engine.begin() as connection:
+        connection.execute(text("""TRUNCATE TABLE recording_release_join"""))
+
+
+def get_releases_for_recording_mbid(connection, recording_mbid):
+    """Returns list of releases for a corresponding recording MBID
+       in recording_release_join table if recording MBID exists else None
+       is returned.
+    """
+
+    result = connection.execute(text("""
+        SELECT release_mbid, release_name
+          FROM recording_release_join
+         WHERE recording_mbid = :recording_mbid
+    """), {"recording_mbid": recording_mbid}
+    )
+
+    if result.rowcount:
```

Comment:
why are you using an if here -- doesnt seem needed, nor do you have it in other places...